.analysis_options

```diff
@@ -6,13 +6,23 @@ analyzer:
 linter:
   rules:
 #    - always_declare_return_types
+#    - always_put_control_body_on_new_line
+    - always_require_non_null_named_parameters
```

Comment:
Sorry, I guess I wasn't clear. I'm not questioning why this specific rule *wasn't* enabled, I'm questioning why any of these rules *are* being enabled.

I think it's great that we have rules to support so much of the style guide! I'm very happy for the work you've been doing. The product is better because of it. Thank you!

But the style guide is not universally accepted among the Dart team, and I think there should be a discussion before we start enabling rules that we have not agreed on. There's a reason that lints are disabled by default. :-)